pinot-query-runtime/src/main/java/org/apache/pinot/query/service/QueryConfig.java

```diff
@@ -22,7 +22,7 @@
  * Configuration for setting up query runtime.
  */
 public class QueryConfig {
-  public static final long DEFAULT_TIMEOUT_NANO = 10_000_000_000L;
+  public static final long DEFAULT_TIMEOUT_NANO = 10_000_000_000_000L;
```

Comment:
(MAJOR) let's not forget to revert this!

Reply:
Ah. yeah. thanks for catching this